docs/issue_template.md

```diff
@@ -0,0 +1,30 @@
+### Step 1: Are you in the right place?
+
+* For general technical questions or "how to" guidance, please search [StackOverflow](https://stackoverflow.com/questions/tagged/puppeteer) with questions tagged "puppeteer" or create a post there.
+* Have an issue or feature request related to the [DevTools Protocol](https://chromedevtools.github.io/devtools-protocol/)? File an issue [here](https://github.com/ChromeDevTools/devtools-protocol/issues/new).
+* Issue with [Headless Chrome](developers.google.com/web/updates/2017/04/headless-chrome)? File on Chromium's [issue tracker](https://bugs.chromium.org/p/chromium/issues/entry?components=Internals%3EHeadless&blocking=705916&cc=skyostil%40chromium.org).
```

Comment:
The link with missing protocol would be erroneously considered relative.

Reply:
done